src/main/server/player/server-player-component.ts

```diff
@@ -13,6 +13,8 @@ import { MathUtil } from '../../client/util/math-util';
 import { PLAYERS } from '../../shared/config/shared-constants';
 import { filter } from 'rxjs/operators';
 
+const LAVA_SAFE_ZONE = 2000;
```

Comment:
It was 20 before so it should be 10 or so.

Reply:
no it was 4000 before, because `(RADIUS * RADIUS * 0.9) = 200 * 200 * 0.9 = 40000 * 0.9 = 36000`

user feedback was that people touched the lava but didnt die, so i reduced it to `2000` to address it

Reply:
i tried 20 or 10, too small

Reply:
I see, you are right. Forgot the square.